cpp/moderncpp.Rmd

```diff
@@ -221,27 +232,34 @@ In this example we cannot convert from `simple_container<int>` to `simple_contai
 
 ## Raw Pointers
 
-There are some caveats with raw pointers, but we don't use raw pointers, so if that ever is the case there are many other resources explaining what is happening.
+There are some caveats with raw pointers (see definition in [ownership](#raw-pointers)), but we _very rarely_ use raw pointers.
+If you're confused about such a case, there are many other resources explaining what is happening.
+
+## How to Read `const` Specifiers
+
+The rule when reading types in C++ is that you should read "from right to left".
+This manifests as `const` applying to the thing immediately to its left.
+If there isn't anything to the left, it applies to the thing on its right - please don't use ["East const"](https://isocpp.org/wiki/faq/const-correctness#const-ref-alt) though...
 
```

Comment:
```suggestion
e.g. `cont int*` and `int const*` are both a pointer to a constant int, `int* const` is a const pointer to an int and `const int* const` or `int const* const` is a const pointer to a const int.
```

Reply:
Resuggesting your suggestion with a typo fix, so I can commit it.
```suggestion
e.g. `const int*` and `int const*` are both a pointer to a constant int, `int* const` is a const pointer to an int and `const int* const` or `int const* const` is a const pointer to a const int.
```